### PR TITLE
fix(admin-ui): Gift Card manage page

### PR DESCRIPTION
**Pull request by kasperkristensen**
**What**
- Updates GC manage page to use product page sections
- Revamps Denomination section
- Updates the location of several components to reflect that they are now shared between the GC and products domain

image
image

Resolves CORE-1089

### CODE REVIEW

File: packages/admin-ui/ui/src/components/forms/product/organize-form/index.tsx
Comment (olivermrbl) [inline packages/admin-ui/ui/src/components/forms/product/organize-form/index.tsx:90]: todo: Looks like this should check should be flipped? 
```suggestion
      categoriesOptions?.length ? (
```

File: packages/admin-ui/ui/src/domain/gift-cards/manage/index.tsx
Comment (olivermrbl) [inline packages/admin-ui/ui/src/domain/gift-cards/manage/index.tsx:54]: todo:
```suggestion
        path="/a/gift-cards"
```

**Review by olivermrbl — CHANGES_REQUESTED** [review]
This is much better - great work @kasperkristensen. Left a couple of todos 👍 

@StephixOne will you give this a review too?

**Review by olivermrbl — APPROVED** [review]
LGTM - pending @StephixOne's approval :)

**Review by StephixOne — APPROVED** [review]
LGTM 👍

**Pull request merged**
